rfc6962-bis.xml

```diff
@@ -1321,7 +1321,10 @@ but it is expected there will be a variety.
 	  </t>
 	  <t hangText="Version">
 	    The version of the protocol supported by the log (currently 1 or 2).
-	  </t>
+    </t>
+    <t hangText="Maximum Chain Length">
+      The longest chain submission this log is willing to accept.
```

Comment:
"Logs SHOULD limit..." is not a MUST, which implies that there needs to be a way to _not_ specify a limit.  Is this done by omitting, or by specifying some magic value for, the "Maximum Chain Length" metadata item?